src/components/pages/TicketsDashboard/TicketsDashboard.js

```diff
@@ -69,20 +69,21 @@ const TicketsDashboard = props => {
   useEffect(() => {
     const getTickets = () => {
       axiosWithAuth.get('/resource-tickets').then(res => {
-        if (res.data.message === null) {
+        if (res.data.length > 0) {
```

Comment:
This is a solid solve - you always want to be sure you know what you're checking for when doing any kind of conditional logic, and the previous implementation made assumptions that simply didn't pan out. Great work!